src/game/Utils/Slider.cc

```diff
@@ -96,7 +96,7 @@ static void SelectedSliderMovementCallBack(MOUSE_REGION* r, UINT32 reason);
 
 SLIDER* AddSlider(UINT8 ubStyle, UINT16 usCursor, UINT16 usPosX, UINT16 usPosY, UINT16 usWidth, UINT16 usNumberOfIncrements, INT8 sPriority, SLIDER_CHANGE_CALLBACK SliderChangeCallback)
 {
-	AssertMsg(guiSliderBoxImage != NULL, "Trying to Add a Slider Bar when the Slider System was never inited");
+	if (!guiSliderBoxImage) InitSlider();
```

Comment:
This could still fail, but then we'd have other problems anyway. Not sure if coverity would complain.